packages/lesswrong/components/posts/UserSuggestNominations.tsx

```diff
@@ -1,33 +1,131 @@
 import React from 'react';
-import { useLocation } from '../../lib/routeUtil';
+import { useLocation, useNavigate } from '../../lib/routeUtil';
 import { registerComponent, Components } from '../../lib/vulcan-lib';
 import { useCurrentUser } from '../common/withUser';
+import Tabs from '@material-ui/core/Tabs';
+import Tab from '@material-ui/core/Tab';
+import {AnalyticsContext} from '@/lib/analyticsEvents'
+import {preferredHeadingCase} from '@/themes/forumTheme'
+import withErrorBoundary from '@/components/common/withErrorBoundary'
+import moment from 'moment'
 
-const UserSuggestNominations = () => {
-    const { SectionTitle, SingleColumnSection, ErrorBoundary, PostsByVoteWrapper } = Components
-    const currentUser = useCurrentUser()
-
-    const { params } = useLocation();
-    
-    // Handle url-encoded special case, otherwise parseInt year
-    const year = ['≤2020', '%e2%89%a42020'].includes(params?.year) ?
-      '≤2020' :
-      parseInt(params?.year)
-    if (!currentUser) return null
-
-    return <ErrorBoundary>
-      <SingleColumnSection>
-        <SectionTitle title={`Your Strong Upvotes from ${year}`}/>
+const styles = (theme: ThemeType): JssStyles => ({
```

Comment:
should not have explicit : JssStyles return type, and classes prop in the component should be typed classes: ClassesType<typeof styles> to get type inference for the available class names